src/main/java/io/github/anthorx/parquet/sql/write/JDBCWriter.java

```diff
@@ -81,4 +83,20 @@ public void write() throws IOException, SQLException {
       }
     }
   }
+
+  private List<RecordField> extractFieldValues(List<String> fieldsNames, Record record) {
+    return fieldsNames
+        .stream()
+        .reduce(new ArrayList<>(), (currentValues, currentFieldName) -> {
+          RecordField result = getNextField(currentFieldName, record);
```

Comment:
I think you can avoid using getNextField function which introduce boilerplate code by using orElse directly :
```
RecordField result = record
             .getField(currentFieldName)
             .orElse(new RecordField<>(currentFieldName, null))
```

Reply:
Yeah, refactored the method and don't see this simplification :/